action.yaml

```diff
@@ -11,6 +11,12 @@ inputs:
     description: 'The name you want to refer to the module to in the Humanitec Platform.'
     required: false
     default: ''
+  tag:
+    description: 'Use tag when you want to bring your own tag'
+    required: false
+  tag_name:
```

Comment:
we use the standard GitHub action convention of using dashes instead of "_"

Also, I think that this name is unclear. I would suggest: `auto-tag` as being more intuitive